crunch-hbase/src/main/java/org/apache/crunch/io/hbase/HFileUtils.java

```diff
@@ -432,6 +518,24 @@ public static void writePutsToHFilesForIncrementalLoad(
     writePutsToHFilesForIncrementalLoad(puts, connection, tableName, outputPath, false);
   }
 
+  public static void writePutsToHFilesForIncrementalLoad(
```

Comment:
javadoc for these 2 new public methods?

Reply:
added, https://github.com/noslowerdna/crunch/commit/9dd88da343727acd4685e220f499f4fc21770dfe